java/memory/src/main/java/org/apache/arrow/memory/NettyAllocationManager.java

```diff
@@ -61,9 +61,14 @@ protected void release0() {
     memoryChunk.release();
   }
 
+  /**
+   * Returns the underlying memory chunk size managed.
+   *
+   * <p>NettyAllocationManager manages memory chunk by the exact requested size.
```

Comment:
is this description accurate? I am not sure but I also lean into explain a bit more:
```suggestion
   * <p>NettyAllocationManager manages memory chunk by computing the allocatedSize based on the requestedSize. Thus the allocatedSize should be returned instead of the initial requestedSize.
```

Reply:
Probably should be:

NettyAllocationManager rounds requested size up to the next power of two.